.github/workflows/main.yaml

```
@@ -0,0 +1,49 @@
+  push:
+    branches:
+      - main
+
+jobs:
+    audit:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v4
+    
+        - name: Set up Go
+          uses: actions/setup-go@v4
```

actionlint (1.7.8)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)